docs/styles.md

```diff
@@ -808,51 +843,67 @@ the side by side content.
 - `os-raise-justify-content-center` centers the grid horizontally.
 - `os-raise-justify-content-between` adds even spacing between content starting after the first element and ending before the last.
 - `os-raise-justify-content-evenly` adds even spacing between content including before the first and after the last element.
+- `os-raise-gap-1` adds 1 rem (16px) of space between content.
+- `os-raise-gap-2` adds 2 rem (32px) of space between content. 
 
 ```html
-<h3>os-raise-d-flex</h3>
-<div class="os-raise-d-flex">
+<h3>os-raise-d-flex-nowrap</h3>
+<div class="os-raise-d-flex-nowrap">
```

Comment:
This has resulted in confusion in the past, and frankly confused me when I looked just now :sweat_smile: , but we should ensure that the HTML and examples are clearly tied together. In this case, the output of the HTML is in a completely different order from the image. Related to some of the other comments, maybe as part of improving the flex related documentation we should break things down very clearly with a 1:1 between image and HTML :man_shrugging: .

Reply:
Good catch - my vote is to break things down more clearly with a 1:1 between the image and HTML. While the flex content can and should be connected (since they only work when paired together), individual images and HTML like we do for other content in the markdown would go a long way.